projects/styles/basic/main.less

```diff
@@ -71,3 +71,15 @@ html,
 ::-ms-reveal {
     display: none;
 }
+
+@supports (-webkit-hyphens: none) {
+    /**
+      * @descriptions:
+      * Safari 16 has bug when for some reason,
+      * the ::after blocks aren't removed
+      * after the destruction of the parent class
+     */
+    :after {
+        content: none;
```

Comment:
I am afraid that it can break something(

Should we use `:where()` (for 0 specificity) ?
Safari 16 supports it
https://caniuse.com/mdn-css_selectors_where

Guys, what do you think about it ?

Reply:
I mean something like this

```css
@supports (-webkit-hyphens: none) {
    /**
      * @descriptions:
      * Safari 16 has bug when for some reason,
      * the ::after blocks aren't removed
      * after the destruction of the parent class
     */
    :where(:after) {
        content: none;
    }
}
```

Reply:
didn't work

Reply:
If it works, we should go for it. Such hack should be as narrow as possible with lowest possible specificity. Also we need to be sure there are no default before/after elements on some native tags that could suffer from this hack.